privacyidea/api/lib/prepolicy.py

```diff
@@ -342,6 +342,37 @@ def papertoken_count(request=None, action=None):
     return True
 
 
+def tantoken_count(request=None, action=None):
```

Comment:
I think this policy is not actually in effect yet?

Reply:
done